tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js

```diff
@@ -131,8 +136,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     fluid.tests.afterHideFunc1 = function () {
```

Comment:
Retrospectively we should take the opportunity to fix namespacing of lots of definitions like this. Even browser tests should be written with the model that one day all these definitions might coexist as part of a single global namespace